### ISSUE TITLE
Covetous coin doesn't seem to work with Tinker's Construct lapis?

### ISSUE THREAD

**Issue by seebs** [BluSunrize/WitchingGadgets#71]
This is sort of speculative, because it's hard to measure, but my general impression has been that a 450 lapis Tinker's Construct tool does indeed have the same Looting III that you'd get from enchantment. However, the Ring of the Covetous Coin seems to work fine on items with the Looting III enchantment per se, but does not seem to work on Tinker's Construct gear. Not sure whether this is even real; my basis is just counting amber from mining a stack or two with a fortune 3 pickaxe and a fortune 3 TiC pickaxe.

**Comment by Entoarox** [comment]
Since I noticed that Beserker's Bracers dont work either I think it simply has something to do with tinkers tools&weapons and how they work.

**Comment by Shahelion** [comment]
Probably because they don't seem to use actual enchantments for the tools. The modifiers affect the tool attributes more directly.

**Comment by Entoarox** [comment]
Actually, Fortune, Looting and Silk touch are all placed on tinkers tools in the proper conditions, so they do use actual enchantments in some cases.

**Comment by catzs** [comment]
No, they use separate code from the Vanilla code to create Silk Touch, Fortune, etc, which is most likely incompatible with this.

**Comment by Entoarox** [comment]
The silk touch, looting and fortune enchantments themselves are placed on the tools when you upgrade them with the jewel/enough lapis, as actual enchantments (Check the NBT), likely done because silk touch & fortune/looting are handled by the blocks & entities themselves, so adding the enchants is how tinkers makes it work, even if the way you get those enchantments on the tools (modifiers) is different.

**Comment by catzs** [comment]
The Tinker's Construct Wikia claims otherwise:
http://tinkers-construct.wikia.com/wiki/Silky_Jewel:
The Silky Jewel is a manufactured modifier item, used to **simulate** the Silk Touch enchantment in TC tools.

**Comment by catzs** [comment]
I'm sorry: https://github.com/SlimeKnights/TinkersConstruct/blob/1.7.10/src/main/java/tconstruct/modifiers/tools/ModLapis.java#L92
It has code to add Fortune to the item, if there is enough Lapis.
